src/me/mrCookieSlime/Slimefun/SlimefunGuide.java

```diff
@@ -716,7 +716,11 @@ public void run(final Player p) {
 							index++;
 						}
 						else {
-							menu.addItem(index, new CustomItem(Material.BARRIER, StringUtils.formatItemName(sfitem.getItem(), false), "", "&rYou do not have Permission", "&rto access this Item"));
+							if (Messages.local.getTranslation("messages.item-permission").get(1).isEmpty()) {
+								menu.addItem(index, new CustomItem(Material.BARRIER, StringUtils.formatItemName(sfitem.getItem(), false), "", Messages.local.getTranslation("messages.item-permission").get(0)));
+							} else {
+								menu.addItem(index, new CustomItem(Material.BARRIER, StringUtils.formatItemName(sfitem.getItem(), false), "", Messages.local.getTranslation("messages.item-permission").get(0), Messages.local.getTranslation("messages.item-permission").get(1)));
```

Comment:
You shouldn't use List Indexing that way here, just pass the whole list, instead of accessing single elements twice.